File: Prueba1Android/www/plugins/BComApp.BComTimer.TimerDelegate/www/TimerDelegate.js
cordova.define("BComApp.BComTimer.TimerDelegate.TimerDelegate", function(require, exports, module) {
               
// __________ Inicia Phonegap __________

var exec = require('cordova/exec');

function TimerDelegate() {}

// TimerDelegate.prototype.initTimer = function() {}
// TimerDelegate.prototype.logout = function() {}
// TimerDelegate.prototype.reinicioTimer = function() {}

TimerDelegate.prototype.resetTimerInactividad = function() {
	exec(
		function (result) { showLoadingLayer(); console.log('resetTimerInactividad success: ' + result); },
    	function (error)  { console.log('resetTimerInactividad error: ' + error);  },
	    "TimerDelegate", 
    	"resetTimerInactividad",
    	[]
  	);
}
               
TimerDelegate.prototype.resetTimerOperacion = function(isWebTrader) {
    exec(
        function (result) { showLoadingLayer(); console.log('resetTimerInactividad success: ' + result); },
        function (error)  { console.log('resetTimerInactividad error: ' + error);  },
        "TimerDelegate",
        "resetTimerOperacion",
        [isWebTrader]
    );
}

var timerDelegate = new TimerDelegate();
module.exports = timerDelegate;

// __________ Termina Phongegap __________

});

File: BTrader-iOS/BCOM/platforms/ios/BTrader/Plugins/BComApp.BComTimer.TimerDelegate/TimerDelegate.m
//
//  TimerDelegate.m
//

#import "TimerDelegate.h"
#import <Cordova/CDV.h>
#import "AppDelegate.h"

@implementation TimerDelegate

- (void)resetTimerInactividad:(CDVInvokedUrlCommand*)command
{
    _callbackInactividad = command.callbackId;
    
    AppDelegate *_appDelegate = [UIApplication sharedApplication].delegate;
    
    [_appDelegate resetInactividadTimer];
}

- (void)resetTimerOperacion:(CDVInvokedUrlCommand*)command
{
    _callbackOperacion = command.callbackId;
    
    BOOL isWebTrader = [[command.arguments objectAtIndex:0] boolValue];
    
    AppDelegate *_appDelegate = [UIApplication sharedApplication].delegate;
    
    [_appDelegate resetOperacionTimer:isWebTrader];
}

- (void) responseInactividad
{
    CDVPluginResult* result = [CDVPluginResult resultWithStatus: CDVCommandStatus_OK messageAsString: @"Reset Timer Inactividad"];
    
    [self.commandDelegate sendPluginResult: result callbackId: _callbackInactividad];
}

- (void) responseOperacion
{
    CDVPluginResult* result = [CDVPluginResult resultWithStatus: CDVCommandStatus_OK messageAsString: @"Reset Timer Operacion"];
    
    [self.commandDelegate sendPluginResult: result callbackId: _callbackOperacion];
}

// - (void)initTimer:(CDVInvokedUrlCommand*)command{}
// - (void)logout:(CDVInvokedUrlCommand*)command{}
// - (void)reinicioTimer:(CDVInvokedUrlCommand*)command{}

@end
